pkg/discovery/worker/aggregation/container_utilization_data_aggregator.go

```diff
@@ -0,0 +1,92 @@
+package aggregation
+
+import (
+	"fmt"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+	"math"
+)
+
+var (
+	maxUtilizationDataStrategy = "maxUtilizationData"
+	allUtilizationDataStrategy = "allUtilizationData"
+
+	DefaultContainerUtilizationDataAggStrategy = maxUtilizationDataStrategy
```

Comment:
As Endre mentioned in yesterday's meeting, I think that the agreement from our earlier meetings was that "all data" would be the default strategy based on preliminary feedback. We can do some testing in our lab between the two strategies and if we find that "max data" is better we can adjust the default, but I think for now we should use the "all data" as the default.